.github/workflows/check-links.yml

```diff
@@ -0,0 +1,17 @@
+name: Check for broken links
+
+on:
+  workflow_dispatch:
+  pull_request:
+    paths:
+      - "source/**"
+
+jobs:
+  check_links:
+    runs-on: ubuntu-latest
+    container:
+      image: ministryofjustice/tech-docs-github-pages-publisher:1.5
```

Comment:
could use v2 here

Reply:
Thanks Nick, will look at this in another PR.